+neurostim/design.m

```diff
@@ -492,7 +492,10 @@ function addAdaptive(o,plg,prm,obj)
                         end
                     else
                         targetFactors = S(2).subs;
-                        if o.nrFactors >0 && (targetFactors{end}~=1 && numel(targetFactors) ~= o.nrFactors && ~(numel(targetFactors)==1 && strcmpi(targetFactors{1},':'))) % allow (:,1) for a one-factor
+                        if o.nrFactors >0 && ( ...
```

Comment:
@cnuahs Thanks so much for this re-organisation and commenting. Will save us all many brain cycles in the future I am sure.